src/utilities/connectors.ts

```diff
@@ -1,13 +1,13 @@
 import { InjectedConnector } from '@web3-react/injected-connector';
 import { WalletConnectConnector } from '@web3-react/walletconnect-connector';
+import { WalletLinkConnector } from '@web3-react/walletlink-connector';
 import { BscConnector } from '@binance-chain/bsc-connector';
 import { LedgerConnector } from '@web3-react/ledger-connector';
 import getNodeUrl from './getRpcUrl';
 
 const POLLING_INTERVAL = 12000;
 const rpcUrl = getNodeUrl();
-// @ts-expect-error ts-migrate(2345) FIXME: Argument of type 'string | undefined' is not assig... Remove this comment to see the full error message
-const chainId = parseInt(process.env.REACT_APP_CHAIN_ID, 10);
+const chainId = parseInt(process.env.REACT_APP_CHAIN_ID || '', 10);
```

Comment:
is it necessary to add empty string?

Reply:
Yeah, otherwise a TypeScript errors is thrown since `process.env.REACT_APP_CHAIN_ID` could be `undefined`, in which case the whole line would create a JS error at runtime since `parseInt` can't be passed an `undefined` value.

Reply:
seems like unassigned chain id should occur an error. shouldn't we handle this case?

Reply:
Maybe default to 97?

Reply:
Yeah assigning the `testnet` ID makes sense. I'll make the update 👍